src/app/public/modules/phone-field/phone-field.component.spec.ts

```diff
@@ -1335,6 +1376,47 @@ describe('Phone Field Component', () => {
         expect(component.phoneControl.valid).toBe(true);
         expect(component.phoneControl.errors).toBeNull();
       }));
+
+      it('should validate properly when a valid number with an extension is given and extensions are allowed', async(() => {
```

Comment:
The guts of these two tests are almost identical to the other two new tests above, as well as to each other, and should be factored into a common function.

Reply:
Good call. This was actually a widespread issue throughout the file. I've factored out a function and implemented it across the spec file.